src/CarbonAware.DataSources/CarbonAware.DataSources.WattTime/src/Configuration/WattTimeClientConfiguration.cs

```diff
@@ -50,5 +51,16 @@ public void Validate()
         {
             throw new ConfigurationException($"{Key}:{nameof(this.BaseUrl)} is not a valid absolute url.");
         }
+
```

Comment:
Is the Validate() method created solely for testing purposes? I dont see it called in the code anywhere.

Reply:
It shouldn't be. The config should be validated when we instantiate the client like in the ElectricityMaps: https://github.com/Green-Software-Foundation/carbon-aware-sdk/blob/a098fd61847e4e2a686de0e69c0a7b2a3327a985/src/CarbonAware.DataSources/CarbonAware.DataSources.ElectricityMaps/src/Client/ElectricityMapsClient.cs#L29

I wonder if it somehow got taken out without us noticing in a merge. I can put it back in

Reply:
Added in